src/game/savegame_bson.c

```diff
@@ -181,10 +252,10 @@ static bool Savegame_BSON_LoadStartInfo(
     for (int i = 0; i < (signed)start_arr->length; i++) {
         struct json_object_s *start_obj = json_array_get_object(start_arr, i);
         if (!start_obj) {
-            LOG_ERROR("Malformed save: invalid start info");
+            LOG_ERROR("Malformed save: invalid discontinued start info");
             return false;
         }
-        START_INFO *start = &game_info->start[i];
+        RESUME_INFO *start = &game_info->start[i];
```

Comment:
For 2.6 / 2.7 saves, the data was stored in `save_info` JSON array. But, START_INFO is now more like current. It's only visual changes, but maybe this should read into `RESUME_INFO *current` and memcpy into start later? That's more in line with what's happening.